cg/server/api.py

```diff
@@ -175,10 +175,10 @@ def parse_families_in_collaboration():
     """Return families in collaboration."""
     customer: Customer = db.get_customer_by_customer_id(customer_id=request.args.get("customer"))
     data_analysis: str = request.args.get("data_analysis")
-    cases_query: Query = db.families(
-        enquiry=request.args.get("enquiry"),
+    cases_query: Query = db.get_filtered_cases(
+        case_internal_id_or_name_search_pattern=request.args.get("enquiry"),
         customers=customer.collaborators,
-        data_analysis=data_analysis,
+        pipeline=data_analysis,
```

Comment:
Maybe out of the scope, we could change data_analysis input  to pipeline as well